### PR TITLE
Added Fgmax devices

### PR DESCRIPTION
**Pull request by rodfferreira**

### CODE REVIEW

**Comment by sjorge** [comment]
FGMAXSW001 is missing the `supports` param so the tests are failing.

**Comment by Koenkk** [comment]
- I've changed `ep` to `l` as the endpoint postfixes as it brings it more in line with other devices
- Could you provide a link to both devices? I cannot find them on google.

Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt/pull/3078

**Comment by rodfferreira** [comment]
Thanks for the reply.

These modules are being tested before commercial release, that's why you could find a link yet.
We are willing to test the integration to Hass with device discovery.

Thank you again.

File: devices.js [hunk 1]
```diff
@@ -8155,6 +8155,32 @@ const devices = [
...
     },
 
+    // FGMAX
+    {
+        zigbeeModel: ['FGMAXSW001'],
+        model: 'FGMAXSW001',
+        vendor: 'FGMAX',
+        description: 'Switch 8 positions',
+        fromZigbee: [fz.on_off_multi_endpoint],
+        toZigbee: [],
```
Comment (Koenkk) [inline devices.js]: I expect it's also possible to control the relays from zigbee right?

**Comment by Koenkk** [comment]
Only devices that are commercially available can be supported by Zigbee2mqtt, until then this cannot be merged.

Anyway, I've updated both PRs to integrate well with zigbee2mqtt so this can already be tested locally with the changes from both PRs.

**Comment by rodfferreira** [comment]
I am sorry, I didn´t know.

Anyway thanks for merging it will help with our tests.

**Pull request closed without merging**